pandas/core/window/indexers.py

```diff
@@ -120,3 +120,23 @@ def get_window_bounds(
             np.zeros(num_values, dtype=np.int64),
             np.arange(1, num_values + 1, dtype=np.int64),
         )
+
+
+class FixedForwardWindowIndexer(BaseIndexer):
+    """Calculate fixed forward-looking rolling window bounds,"""
+
+    @Appender(get_window_bounds_doc)
+    def get_window_bounds(
+        self,
+        num_values: int = 0,
+        min_periods: Optional[int] = None,
```

Comment:
this should raise if min_periods, center, closed are not None

Reply:
Implemented it a bit differently, pinged you in the comment on the implementation.